.github/actions/bazel/bin/bazel

```diff
@@ -39,19 +39,19 @@ done
 
 log "Bazel wrapper starting"
 
-# All args, including the command
-bazel_args=( "$@" )
-
 # We've asserted there are no startup options, so this must be the command (like "build" or "run")
-# stored here again for easy access
 bazel_command="$1"
+shift
+
+# Extra args we'll be injecting
+bazel_args=( )
```

Comment:
this is not ideal because bash <4 will consider it as empty, though seems to work even on our intel runners...